Mathlib/Order/ConditionallyCompleteLattice/Basic.lean

```diff
@@ -296,6 +296,14 @@ instance (α : Type*) [ConditionallyCompleteLinearOrder α] : ConditionallyCompl
 
 end OrderDual
 
+theorem toDual_ciSup [ConditionallyCompleteLattice α] (f : ι → α) :
+    OrderDual.toDual (⨆ i, f i) = ⨅ i, (OrderDual.toDual ∘ f) i :=
+  rfl
```

Comment:
This lemma should assume `[SupSet α]`, same for the next one. Do we use `ciSup` for lemmas like this? I thought that this is for conditionally complete lattice versions of lemmas that have `iSup` versions with fewer assumptions.

Reply:
Done, and made sure to generalize the existing lemmas out of `CompleteLattice` to `SupSet`.